automatic/selenium.powershell/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,51 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = Split-Path -Parent $MyInvocation.MyCommand.Definition
+
+$ModuleName = "Selenium"
+$ModuleVersion = $env:ChocolateyPackageVersion
+
+$SourcePath = Join-Path $toolsDir "$($ModuleName).zip"
+$SavedPaths = Join-Path $ToolsDir 'installedpaths'
```

Comment:
Please keep the case consistent, as it is defined with a lowercase `t`, please use that same casing everywhere.

```suggestion
$SourcePath = Join-Path $toolsDir "$($ModuleName).zip"
$SavedPaths = Join-Path $toolsDir 'installedpaths'
```